components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -124,10 +122,26 @@
         },
         sidecars: sidecars,
       };  // buildTemplate
-
-      local buildImageTemplate(step_name, image, base_image, tf_package) =
-        buildTemplate(
-          step_name,
+      local buildImageTemplate(tf_version, device) = {
+        local image = params.registry + "/tensorflow-notebook-" + tf_version + "-" + device + ":" + params.versionTag,
```

Comment:
nit. "tensorflow-" + tf_version + "-notebook"

I think this makes it more clear that tf_version applies to tensorflow

Reply:
Done